### PR TITLE
Fix/inspector rearrange

### PR DESCRIPTION
**Pull request by OmarDaSilva**
Fixes #[892]

**Problem:**
Inspector position and dimension-related properties are not at the top

**Fix:**
Moved the component containing these properties to the top

### CODE REVIEW

**Comment by OmarDaSilva** [comment]
Can you please review

File: editor/src/components/inspector/sections/style-section/style-section.tsx [hunk 1]
```diff
@@ -8,6 +8,14 @@ import { ShadowSubsection } from './shadow-subsection/shadow-subsection'
 import { TextShadowSubsection } from './text-subsection/text-shadow-subsection'
 import { AutosizingTextSubsection } from './text-subsection/text-subsection'
 import { TransformSubsection } from './transform-subsection/transform-subsection'
+import {
```
Comment (balazsbajorics) [inline editor/src/components/inspector/sections/style-section/style-section.tsx:11]: it seems like you don't need these imports – probably leftover from previous commits

**Comment by maltenuhn** [comment]
Mea culpa for this one, moving the position & layout props isn't a "changed my mind". I simply missed it last week.

File: editor/src/components/inspector/sections/style-section/style-section.tsx (hunk 1 above)
Comment (OmarDaSilva) [reply editor/src/components/inspector/sections/style-section/style-section.tsx]: Done!

**Review by Rheeseyb — APPROVED** [review]

**Review by balazsbajorics — APPROVED** [review]

**Review by enidemi — APPROVED** [review]

**Pull request merged**